base/src/FrameContainerQueue.cpp

```diff
@@ -39,6 +39,11 @@ void FrameContainerQueue::clear()
 	return bounded_buffer<frame_container>::clear();
 }
 
+void FrameContainerQueue::flush()
+{
+	return bounded_buffer<frame_container>::flush();
+}
+
```

Comment:
none of these functions are needed if we are just calling the base class function.  all of them can be removed from FrameContainerQueue.h and cpp.